packs/prometheus/variables.hcl

```diff
@@ -109,6 +119,22 @@ scrape_configs:
 EOF
 }
 
+variable "prometheus_task_app_rules_yaml" {
+  description = "The Prometheus configuration to pass to the task."
```

Comment:
Can we update this to be more descriptive as I believe this is specific to alerting rules?

Reply:
it is very specific to alerting rules, i ll update

Reply:
updated as requested